### PR TITLE
More async await in actions

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/actions/wallets.tsx [hunk 1]
```diff
@@ -1643,22 +1652,25 @@ function* loadStaticConfig(state: TypedState, action: ConfigGen.DaemonHandshakeP
...
+    const res: Saga.RPCPromiseType<
+      typeof RPCStellarTypes.localGetStaticConfigLocalRpcPromise
+    > = yield RPCStellarTypes.localGetStaticConfigLocalRpcPromise()
+    yield Saga.put(
+      WalletsGen.createStaticConfigLoaded({
+        staticConfig: I.Record(res)(),
+      })
+    )
+  } finally {
+    yield Saga.put(
```
Comment (songgao) [inline shared/actions/wallets.tsx]: Did you mean to move this into `finally`?

File: shared/actions/platform-specific/index.desktop.tsx [hunk 2]
```diff
@@ -234,11 +231,16 @@ const onOutOfDate = (
...
+const prepareLogSend = async (
+  _: Container.TypedState,
+  action: EngineGen.Keybase1LogsendPrepareLogsendPayload
+) => {
   const response = action.payload.response
-  dumpLogs().then(() => {
+  try {
+    await dumpLogs()
+  } finally {
     response && response.result()
```
[the remark this answers is not in the document]
Comment (chrisnojima) [reply shared/actions/platform-specific/index.desktop.tsx]: result is responding to the rpc. it says we're done dumping the logs. we don't return an action cause there's no action side effect

File: shared/actions/wallets.tsx (hunk 1 above)
Comment (chrisnojima) [reply shared/actions/wallets.tsx]: yes, these calls where we call handshakeWait true/false should always be balanced. else if it throws you'll be stuck forever

**Pull request merged**